gatsby_fp/src/components/Layout/Body/Fields/GenInfoTable.js

```diff
@@ -67,12 +67,7 @@ export default function GenInfoTable({
             ></p>
           )}
           {phone && (
-            <p
-              valign="top"
-              dangerouslySetInnerHTML={{
-                __html: `Phone: ${phone}`,
-              }}
-            ></p>
+            <p valign="top">Phone: <a href={phone[0].url}>{phone[0].human_readable}</a></p>
```

Comment:
@mf-stevens Do we need it to be a link?

Reply:
In my opinion, anything that can be linked should be. Otherwise we are making people copy/paste, or manually re-type, etc. and I see no value in that.

Reply:
Agreed, I asked because the one in prod is not.

Reply:
Simple enough to remove if someone makes a case why we should.